cypress/support/Train.js

```diff
@@ -351,7 +352,7 @@ export function VerifyChatTurnIsAnExactMatch(expectedTurnText, expectedTurnCount
 function VerifyChatTurnInternal(expectedTurnCount, turnIndex, doVerification) {
   cy.WaitForStableDOM()
   let chatMessages
-  cy.wrap(undefined).should(() => { 
+  cy.wrap(1).should(() => { 
```

Comment:
It turns out that using `undefined` here prevents Cypress from invoking their retry logic, which was the whole point of wrapping it like I did. I noticed that while debugging changes to the `SaveAsIsVerifyInGrid` function.